apps/desktop/src/renderer/routes/_authenticated/_dashboard/pull-requests/$prNumber/components/PullRequestCommentThread/comment-thread.css

```
@@ -0,0 +1,231 @@
+	/* Full-bleed within the visible code column, flush with the diff pane's
+	 * edges. The bubble is slotted into a shrink-to-fit flex item
+	 * ([data-annotation-content]), so a plain 100% resolves against an
+	 * indefinite width and lets wide content overflow the pane — cqi queries
+	 * the [data-line-annotation] cell instead (made a container in
+	 * useDiffCodeViewTheme's unsafeCSS). */
+	width: 100%;
+	max-width: 100cqi;
+	min-width: 0;
+	box-sizing: border-box;
+	overflow-wrap: anywhere;
+	word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)